client/jetpack-connect/controller.js

```diff
@@ -107,6 +118,7 @@ const getPlanSlugFromFlowType = ( type, interval = 'yearly' ) => {
 };
 
 export function redirectWithoutLocaleIfLoggedIn( context, next ) {
+	debug( 'controller: redirectWithoutLocaleIfLoggedIn', context.params );
```

Comment:
I'm not completely sure what the `debug()` function is but I'm assuming it's similar to a console.log for debugging purposes and we'll want to remove them before merging into production. 👍

Reply:
We actually want to keep them so that we can debug in prod if need be. For instance, you can reveal the tracked events in your browser console by running `localStorage.setItem( "debug", "calypso:analytics" );` and reloading the page.

Reply:
It's a very useful tool. I added them because I wanted to see what controllers were being used when I tested different URLs.

Reply:
Ahh I see.. Yes, makes sense, awesome, thanks @monsieur-z!  😄